Dockerfile.online

```diff
@@ -0,0 +1,70 @@
+ARG EE_BASE_IMAGE=registry.redhat.io/ansible-automation-platform-20-early-access/ee-minimal-rhel8
```

Comment:
What if we named this Dockerfile.base, and then had the Dockerfile.offline file be built off of this?

Reply:
Will take care of this in a future Jira